drivers/hmis/app/graphql/types/application/enrollment_access_summary.rb

```diff
@@ -0,0 +1,59 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class Application::EnrollmentAccessSummary < Types::BaseObject
+    # maps to Hmis::EnrollmentAccessSummary
+    graphql_name 'EnrollmentAccessSummary'
+    field :id, ID, null: false
+    field :last_accessed_at, GraphQL::Types::ISO8601DateTime, null: false
+
+    # The project/enrollment may be marked as deleted. Expose only name and ID since deleted objects may not work with
+    # our resolvers. Project and client are nullable since associated records could be completely missing
+    field :enrollment_id, ID, null: false
+
+    field :client_name, String, null: true
+    field :client_id, ID, null: true
+
+    field :project_id, ID, null: true
+    field :project_name, String, null: true
+
+    def project_id
+      project&.id
+    end
+
+    def project_name
+      project&.project_name
+    end
+
+    def client_id
+      client&.id
+    end
+
+    def client_name
+      client&.brief_name
+    end
+
+    available_filter_options do
+      arg :search_term, String
+      arg :on_or_after, GraphQL::Types::ISO8601Date
+    end
+
+    protected
+
+    def enrollment
+      load_ar_association(object, :enrollment, scope: Hmis::Hud::Enrollment.with_deleted)
+    end
+
+    def project
+      load_ar_association(enrollment, :project, scope: Hmis::Hud::Project.with_deleted) if enrollment
```

Comment:
This will be nil for WIP Enrollments. We need to do something similar to how we load projects on the Enrollment type:

```
    def project
      if object.in_progress?
        wip = load_ar_association(object, :wip)
        load_ar_association(wip, :project)
      else
        load_ar_association(object, :project)
      end
    end
```

Reply:
Good catch, thanks! I'll fix taht.

Something to note is that it looks like Hmis::Wip does not use soft-deletion despite having a deleted_at column. Perhaps it should though?

```
> Hmis::Wip.paranoid?
=> false
> Hmis::Wip.column_names.grep /delete/
=> ["deleted_at"]
```

Reply:
I decided to add soft-deletion the wip model. There's the possibility this causes regressions if we are using the wip model without the default scope somewhere

Reply:
Ok, thanks. We shouldn't be.